zerver/actions/invites.py

```diff
@@ -36,10 +36,13 @@
 from zerver.tornado.django_api import send_event
 
 
-def notify_invites_changed(realm: Realm) -> None:
+def notify_invites_changed(realm: Realm, *, acting_user: Optional[UserProfile] = None) -> None:
     event = dict(type="invites_changed")
     admin_ids = [user.id for user in realm.get_admin_users_and_bots()]
-    send_event(realm, event, admin_ids)
+    recipient_ids = admin_ids
+    if acting_user and acting_user.id not in recipient_ids:
+        recipient_ids.append(acting_user.id)
```

Comment:
This logic does not match the API documentation -- we're notifying the **acting user**, not the user who created the modified invitation. I'm pretty sure this implementation is thus wrong if say an administrator revokes an invitation that a normal user had created.

This is definitely a bug and we should fix it but it's also obscure enough that I'm not going to block merging this on doing so; we can just fix it in a near-term follow-up.

Reply:
I think there is nothing wrong. We are passing the user who created the invitation not the one changing the invitation to `acting_user`. We must change the `acting_user` to some other name to avoid this confusion.

Reply:
Yeah, I think we can just rename the `acting_user` to something else. I was thinking of `referred_by` first, but that would also be confusing since we do not include which invite wass changed in the event. Probably something like `changed_invite_referrer` could be better.

Reply:
Yeah `changed_invite_referrer` works for me.